lib/runtime/wazero/instance_test.go

```diff
@@ -38,6 +38,9 @@ import (
 //go:embed testdata/parachain.yaml
 var parachainTestDataRaw string
 
+//go:embed testdata/parachain2.yaml
+var parachain2TestDataRaw string
+
```

Comment:
could you change to parachains_configuration_v180.yaml? then we know what is the purpose of the yaml file